src/organisation/store/reducers/organisation.reducer.ts

```diff
@@ -37,6 +35,31 @@ export function reducer(
 
     }
 
+    case fromOrganisation.UPDATE_ORGANISATION_PBA_PENDING_ADD: {
+      const organisationDetails = new Organisation(state.organisationDetails);
+      const pendingAddPbaNumbers = action.payload as string[];
+      const uniquePendingPba = pendingAddPbaNumbers.filter((i, p, s) => s.indexOf(i) == p);
```

Comment:
could we give these i, p, s more desciptive names?

Reply:
done